src/fetch/transfer_file.py

```diff
@@ -107,6 +107,7 @@ def auto_propose(
         payout_transfers = dune.get_transfers()
 
     Transfer.sort_list(payout_transfers)
+    payout_transfers = filter(lambda payout: payout.amount_wei > args.min_transfer_amount_wei, payout_transfers)
```

Comment:
If you are getting line to long error on the linter, you can run `make check` and it will auto-format this line.

Reply:
Right, I thought about changing this, but I think it would add more complexity and realistically we don't want to reduce the CoW amount (at least for now) as it's always at least 37 CoW (maybe this changes with CiP-20 though)